lms/templates/student_account/account_settings.html

```diff
@@ -32,7 +32,7 @@
 
 <%block name="js_extra">
 <%static:require_module module_name="js/student_account/views/account_settings_factory" class_name="AccountSettingsFactory">
-    var isShibAuth = ${ json.dumps(is_shib_auth) };
+    var isShibAuth = ${ is_shib_auth | n, dump_js_escaped_json };
```

Comment:
Why this change?

Reply:
Keeping up-to-date with upstream's changes to the other subsequent vars